pgbouncer-mixin/panels.libsonnet

```diff
@@ -0,0 +1,156 @@
+            Average duration of queries being processed by PgBouncer.
+          |||
+        )
+        + g.panel.timeSeries.standardOptions.withUnit('ms'),
+
+      networkTraffic:
+        commonlib.panels.network.timeSeries.traffic.new(
+          'Network traffic',
+          targets=[t.networkTrafficRecieved, t.networkTrafficSent],
+          description=|||
+            Volume in bytes of network traffic received by PgBouncer.
+          |||
+        )
+        + g.panel.timeSeries.standardOptions.withUnit('Bps')
+        + g.panel.timeSeries.options.legend.withDisplayMode('table')
+        + g.panel.timeSeries.options.legend.withPlacement('right'),
+
+      transactionRate:
+        commonlib.panels.generic.timeSeries.base.new(
+          'SQL transaction rate',
+          targets=[t.transactionRate],
+          description=|||
+            Rate of SQL transactions pooled.
+          |||
+        )
+        + g.panel.timeSeries.standardOptions.withUnit('txn/s'),
+
+      transactionAverageDuration:
+        commonlib.panels.generic.timeSeries.base.new(
+          'SQL average transaction duration / $__interval',
+          targets=[t.transactionAverageDuration],
+          description=|||
+            Average duration of SQL transactions pooled.
+          |||
+        )
+        + g.panel.timeSeries.standardOptions.withUnit('ms'),
+
+      serverConnections:
+        commonlib.panels.generic.timeSeries.base.new(
+          'Server connections',
+          targets=[t.serverIdleConnections, t.serverUsedConnections, t.serverLoginConnections, t.serverTestingConnections],
+          description=|||
+            Number of various server connection states.
+          |||
+        )
+        + g.panel.timeSeries.standardOptions.withUnit('conn')
+        + g.panel.timeSeries.options.legend.withDisplayMode('table')
+        + g.panel.timeSeries.options.legend.withPlacement('right'),
+
+      granularActiveClientConnections:
+        commonlib.panels.generic.timeSeries.base.new(
+          'Active client connections',
+          targets=[t.granularActiveClientConnections],
+          description=|||
+            Current number of active client connections.
+          |||
+        )
+        + g.panel.timeSeries.standardOptions.withUnit('conn'),
+
+      clientsWaiting:
+        commonlib.panels.generic.timeSeries.base.new(
+          'Waiting clients',
+          targets=[t.clientsWaiting],
+          description=|||
+            Current number of client connections waiting on a server connection.
+          |||
+        )
+        + g.panel.timeSeries.standardOptions.withUnit('clients'),
+
+      maxClientWaitTime:
+        commonlib.panels.generic.timeSeries.base.new(
+          'Max client wait time',
+          targets=[t.maxClientWaitTime],
+          description=|||
+            Age of the oldest unserved client connection in seconds.
+          |||
+        )
+        + g.panel.timeSeries.standardOptions.withUnit('s'),
```

Comment:
usually for all these timeseries we don't show the cluster in the legend right?

Reply:
I guess, if we decide to make cluster a single select on this, we wouldn't show it. I'll change this